F21/abeiler7/to_lowercase.py

```diff
@@ -0,0 +1,30 @@
+import re
+import unicodedata
+
+
+def to_lower(word, lang):
+    case_insensitive_langs = r'(zh)|(ja)|(th)'
+    if not re.compile(case_insensitive_langs).match(lang):
+        results = language_select(word, lang)
+    else:
+        results = word
+
+    return results
+
+def language_select(word, lang):
+    updt_word = word
+    if re.compile("tr*").match(lang) or re.compile("az*").match(lang):
+        updt_word = word.replace('I','\u0131')
+    elif re.compile("ga*").match(lang):
+        if re.search(r'\b([nt])([\u00C1\u00C9\u00CD\u00D3\u00DA])', word):
```

Comment:
Good idea. I moved the irish regexp to a variable since it is used multiple times. I also went ahead and added comments to define each of the unicode characters that I am using which should also help readability.